api/inspector_test.go

```diff
@@ -33,16 +32,17 @@ func TestInspectorPeerStreams(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	baseAddress := network.NewBzzAddr(baseKey, baseKey)
```

Comment:
I agree, I started to change this on the whole project but did not push yet :) Thanks!

Reply:
Actually, this concrete case needs underlay address because of the `PeerInfo()` function, so I have decided to create another rand key for peer base address. I know it's not needed, but I think it gives more info in tests about the nature of address. WDYT?

Reply:
Or maybe just to add a comment justifying the usage of baseKey in this particular case. That would be also good, at least for me.